common/templates/partials/header.njk

```diff
@@ -7,7 +7,7 @@
 
     <div class="moj-header__logo">
 
-      {{ renderMojCrest() }}
+      {{ renderMojCrest(crestColour, 40) }}
```

Comment:
Not sure if I'm missing where `crestColour` is defined, if not I assume it's masking `undefined` which would then default in the macro?

Reply:
Yep - I'll sort this.

Reply:
👍